packages/yoco/.gitignore

```diff
@@ -0,0 +1,5 @@
+dist/
+gh-pages/
+docs/
```

Comment:
out of curiosity really, can we combine lines 3-5

Reply:
I'm not aware of a way to do that.

In case you're asking because you're unaware of what it does, this ignores the folder besides the two files I mentioned

Reply:
i guess you mean except the two files, but no, i assumed and checked to verify 
was just curious if there was a shorter one-liner for it, but apparently not, that's the recommended method